moz_logging/sandboxes/heka/analysis/moz_logging_ingestion_error_monitor.lua

```diff
@@ -24,36 +25,59 @@ alert = {
     email = {recipients = {"trink@mozilla.com"}},
   },
   thresholds = {
-    -- ["input.bastion_systemd_sshd"] = {inactivity_timeout = 60, percent = 0.5} -- a timeout of 60 or more disables the check as the alert window is only one hour
-    _default_ = {inactivity_timeout = 5, -- minutes percent = 0.5} -- if not specified the default is no monitoring
+    -- ["<input_name>"] = {
+        -- percent            - required (0 - 100), maximum percent error before alerting
+        -- inactivity_timeout - optional (1 - 3600 seconds), time with no data before alerting
+           -- rounded up to the nearest minute; specified in seconds to keep the units consistent
+        -- latency            - optional (1 - 3600 seconds), median latency before alerting
+    -- },
+    ["*"] = { -- if not specified the default is no monitoring
+        percent = 0.5,
+        inactivity_timeout = 300,
+        latency = 60
+    }
   }
 }
 ```
 --]]
-_PRESERVATION_VERSION = 0
+_PRESERVATION_VERSION = (read_config("preservation_version") or 0) + 1
```

Comment:
This seems novel. Maybe add a comment explaining it.

Reply:
This isn't documented anywhere: I will add it to http://mozilla-services.github.io/lua_sandbox/sandbox.html also when looking for examples I came across a couple instances where this syntax is not being used, they will be upated.